docs/locale/ja/LC_MESSAGES/logging.po

```diff
@@ -2,64 +2,76 @@
 # Copyright (C) 2015-2017, Rapptz
 # This file is distributed under the same license as the discord.py package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, 2018.
-#
+# 
 #, fuzzy
 msgid ""
 msgstr ""
 "Project-Id-Version: discord.py 1.0.0a\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2018-07-31 14:21-0400\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"Last-Translator: Episword <mistio100@gmail.com>, 2018\n"
+"Language-Team: Japanese (Japan) (https://www.transifex.com/discord-py/teams/88924/ja_JP/)\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.5.3\n"
+"Language: ja_JP\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
 
 #: ../../logging.rst:5
 msgid "Setting Up Logging"
-msgstr ""
+msgstr "ログの設定"
 
 #: ../../logging.rst:7
 msgid ""
-"*discord.py* logs errors and debug information via the :mod:`logging` "
-"python module. It is strongly recommended that the logging module is "
-"configured, as no errors or warnings will be output if it is not set up. "
-"Configuration of the ``logging`` module can be as simple as::"
+"*discord.py* logs errors and debug information via the :mod:`logging` python"
+" module. It is strongly recommended that the logging module is configured, "
+"as no errors or warnings will be output if it is not set up. Configuration "
+"of the ``logging`` module can be as simple as::"
 msgstr ""
+"*discord.py*はPythonの :mod:`logging` "
```

Comment:
RST doesn't like emphasis marks without a space (and produces a warning about them on build):
```diff
- *discord.py*はPythonの :mod:`logging`
+ *discord.py* はPythonの :mod:`logging`
```